File: cpp/fizzbuzz.cpp
//
// Created by Henry Heng on 15/01/2026.
//
#include <iostream>
using namespace std;

int main() {
    int X, Y, N;
    cin >> X >> Y >> N;

    for (int i = 1; i <= N; i++) {
        if (i != 1) cout << endl;
        if (i % X == 0 && i % Y == 0) cout << "Fizzbuzz";
        else if (i % X == 0) cout << "Fizz";
        else if (i % Y == 0) cout << "Buzz";
        else cout << i;
    }
    return 0;
}

File: python/fizzbuzz.py
x = y = n = 0

while not 1 <= x < y <= n <= 100:
	user_input = input()
	if len(user_input.split()) != 3:
		continue
	[x, y, n] = [int(x) for x in user_input.split()]

for i in range(1, n+1):
	if i % x == 0 and i % y == 0:
		print("FizzBuzz")
	elif i % x == 0:
		print("Fizz")
	elif i % y == 0:
		print("Buzz")
	else:
		print(i)
